packages/components/src/ui/styles/components/README.md

```diff
@@ -0,0 +1,34 @@
+# Components
+
+## StyleFrameProvider
```

Comment:
This is great but I wonder if we could achieve the same without another provider, I mean why can't the style system directly relies on `ownerDocument` or something instead of requiring this provider.

Reply:
I'm not sure I follow. Can you elaborate more on your alternative solution to iframe handling?

Reply:
I don't know the system enough how the current system works to propose anything meaningful. I guess my thinking is why can't a component just inject its own styles into the frame where it's used. This is just a random thought now, not important for this PR.